src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PersonRow } from './PersonRow';
+
+export const PeopleTable:React.FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
loading is true already, you don't have to set it again 😉

Reply:
yeah i was same in advanvced version and copied to adv before changed :P its done :P